documentation/docs/article/guides/rails_integration_with_null_driver.md

```diff
@@ -84,3 +84,62 @@ describe 'example', driver: :null do
   end
 end
 ```
+
+## Minitest Usage
+
+We can do something similar with the default Rails setup using Minitest. Here's the same example written with Minitest:
+
+```rb
+# test/application_system_test_case.rb
+
+require 'playwright'
+
+class CapybaraNullDriver < Capybara::Driver::Base
+  def needs_server?
+    true
+  end
+end
+
+Capybara.register_driver(:null) { CapybaraNullDriver.new }
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :null
+
+  def self.playwright
+    @playwright ||= Playwright.create(playwright_cli_executable_path: 'npx playwright')
+  end
+
+  def initialize(...)
+    super(...)
+    base_url = Capybara.current_session.server.base_url
+    @playwright_browser = self.class.playwright.playwright.chromium.launch(headless: false)
+    @playwright_page = @playwright_browser.new_page(baseURL: base_url)
```

Comment:
I don't know much about Minitest on Rails, but ActiveSupport::TestCase provides `setup { ... }` and `teardown { ... }` callback methods. Isn't it better to use them?
Could you let me know your opnion about this?

It seems Page must be created for each testcase as RSpec example shows.

Reply:
So this is typical for most apps. I chose to go with `initializer` and `after_teardown` because it would mean when users do make a setup method, they need to make sure to call super() which with Capybara they don't currently.

```rb
def setup
  super # browser isn't setup until we call super
end
```

Reply:
> when users do make a setup method, they need to make sure to call super() which with Capybara they don't currently.

I got it. Thank you.

How about using `before_setup` instead of `initialize`, as ActiveRecord::TestFixtures actually does?
https://github.com/rails/rails/blob/06198ed7ad822145e24d9aa964aba97bdb0f8b39/activerecord/lib/active_record/fixtures.rb#L711

```ruby
def before_setup
  @playwright_browser = playwright.chromium.launch(headless: false)
  base_url = Capybara.current_session.server.base_url
  @playwright_page = @playwright_browser.new_page(baseURL: base_url)
  super
end

def after_teardown
  super
  @playwright_browser.close
end
```

Reply:
Seems reasonable to me!